pymodbus/repl/server/main.py

```diff
@@ -180,7 +180,6 @@ def run(
     modbus_config = modbus_config.get(modbus_server, {})
     modbus_config = process_extra_args(extra_args, modbus_config)
     if modbus_server != "serial":
-        modbus_port = int(modbus_port)
```

Comment:
Removing this causes problems, because the parameter is read as a str and the lower layers expect an int.